AspNetCore.sln

```diff
@@ -1786,6 +1786,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Authen
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "IdentitySample.ApiEndpoints", "src\Identity\samples\IdentitySample.ApiEndpoints\IdentitySample.ApiEndpoints.csproj", "{37FC77EA-AC44-4D08-B002-8EFF415C424A}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Components.Endpoints.Performance", "src\Components\Endpoints\perf\Microsoft.AspNetCore.Components.Endpoints.Performance.csproj", "{642A8E5C-F2FA-4A4D-B496-B649657ED8AB}"
```

Comment:
Please follow the naming conventions of [other](https://github.com/dotnet/aspnetcore/blob/main/src/Mvc/perf/Microbenchmarks/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.Microbenchmarks.csproj) benchmark projects.
https://github.com/dotnet/aspnetcore/blob/97acf9de12b3d724a4ea8072424b889352969c42/Directory.Build.props#LL25C78-L25C78

Reply:
@BrennanConroy can you clarify what convention? Components already has a project and I was following the same naming convention: https://github.com/dotnet/aspnetcore/blob/main/src/Components/Components/perf/Microsoft.AspNetCore.Components.Performance.csproj